hazelcast/src/test/java/com/hazelcast/test/starter/HazelcastVersionLocator.java

```diff
@@ -87,10 +88,10 @@ private static void downloadArtifact(Artifact artifact, String version) {
         ProcessBuilder builder = new ProcessBuilder(buildMavenCommand(artifact, version).split(" "))
                 .inheritIO();
         try {
-            int code = builder.start().waitFor();
-            checkState(code == 0, "Maven dependency:get failed");
```

Comment:
What do you think about checking `process.exitValue()==0` as well?

Reply:
Good idea, added